torch/fx/node.py

```diff
@@ -45,6 +45,7 @@
     _ops.aten._assert_async.msg,
     _ops.aten._assert_scalar.default,
     _ops.aten.copy_.default,
+    _ops.aten._foreach_div,
```

Comment:
`_ops.aten._foreach_div` is not in-place IIRC. Any chance you meant `_ops.aten._foreach_div_`?

Reply:
This was a mistake. We don't need this and I'll remove it in the next update. Thanks!